### PR TITLE
Prevents the user from changing registration tabs while the current t…

### PR DESCRIPTION
**Pull request by edrisse**
fix #287

### CODE REVIEW

File: app/registration/controllers/patientCommonController.js [hunk 1]
```diff
@@ -5,23 +5,37 @@
...
 
     // TODO: Remove dependency on $scope!
     var vm = this;
+    console.info(vm);
     vm.patient = $scope.patient;
     vm.patientAttributes = [];
     vm.patientIdentifierTypes = [];
     vm.srefPrefix = $scope.srefPrefix;
     vm.today = dateUtil.getDateWithoutTime(dateUtil.now());
+    vm.func = function (form, state) {
```
Comment (ynurmahomed) [inline app/registration/controllers/patientCommonController.js]: Looks like this is not being used.

File: app/registration/controllers/patientCommonController.js [hunk 2]
```diff
@@ -5,23 +5,37 @@
...
   function PatientCommonController($scope, $http, $state, patientAttributeService, patientService, localStorageService,
-                                   spinner, notifier, $filter) {
+                                   spinner, notifier, $filter, TabManager) {
 
     var dateUtil = Bahmni.Common.Util.DateUtil;
     var patientConfiguration = $scope.patientConfiguration;
 
     // TODO: Remove dependency on $scope!
     var vm = this;
+    console.info(vm);
```
Comment (ynurmahomed) [inline app/registration/controllers/patientCommonController.js]: I think this is only useful for debugging.

**Comment by larslemos** [comment]
Its @edrisse first PR. Good and clean.  @
@edrisse Welcome to the "combat line"
Later we can find a way to avoid multiple messages 
image

**Comment by edrisse** [comment]
Thanks for the help and the patience guys

**Pull request merged**